### DISCUSSION TITLE
Superfluous double quote in html.tpl

### DISCUSSION THREAD

**Discussion by JanMosigItemis** [aquasecurity/trivy#5033] [Bugs]
### Description

Hi there,

html.tpl contains a superfluous double quote. Doesn't seem to break functionality though: https://github.com/aquasecurity/trivy/blob/main/contrib/html.tpl#L131

Will provide PR.

Regards

### Desired Behavior

html.tpl contains valid markup.

### Actual Behavior

https://github.com/aquasecurity/trivy/blob/main/contrib/html.tpl#L131

![image](https://github.com/aquasecurity/trivy/assets/16935994/59e77cf6-8bb1-4799-9707-9c420e726616)

### Reproduction Steps

```bash
N/A
```

### Target

None

### Scanner

None

### Output Format

Template

### Mode

Standalone

### Debug Output

```bash
N/A
```

### Operating System

Linux

### Version

```bash
0.44.1
```

### Checklist

- [X] Run `trivy image --reset`
- [X] Read [the troubleshooting](https://aquasecurity.github.io/trivy/latest/docs/references/troubleshooting/)

**Comment by knqyf263** [comment]
Thanks for reporting. Created https://github.com/aquasecurity/trivy/issues/5034
